recipes-app/node-red-node-serialport/node-red-node-serialport_1.0.1.bb

```diff
@@ -8,13 +8,12 @@
 # COPYING.MIT file in the top-level directory.
 #
 
+inherit node-red-module
 inherit npm
 
 DESCRIPTION = "Node-RED nodes to talk to serial ports"
 
-NPM_LOCAL_INSTALL_DIR = "/root/.node-red"
-
 do_prepare_build_append() {
-    sed -i '/override_dh_install:/a\\trm -r ${PP}/image/root/.node-red/node_modules/node-red-node-serialport/node_modules/@serialport/bindings-cpp/prebuilds' \
+    sed -i '/override_dh_install:/a\\trm -r ${PP}/image/${@ d.getVar("NPM_LOCAL_INSTALL_DIR") or "/usr/lib" }/node_modules/node-red-node-serialport/node_modules/@serialport/bindings-cpp/prebuilds' \
```

Comment:
We don't have a fallback for undefined NPM_LOCAL_INSTALL_DIR elsewhere, why here?

Reply:
Ah yes... This is not required anymore. I'll remove it in the next version.